### PR TITLE
Enable NMEA GST message support

### PR DESCRIPTION
**Pull request by anth-cole**
Bring in an updated gnss-converters and enable the GST message

@benjaminaltieri

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.1.0-develop-2019032221-3-g43895e54

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend runs](https://gnss-analysis.swiftnav.com/jobs/id=967529)
+ [live-roof-650-townsend-dropouts-zero-baseline runs](https://gnss-analysis.swiftnav.com/jobs/id=967530)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019032221-3-g43895e54&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019032221-3-g43895e54&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019032221-3-g43895e54&build_type=buildroot_pr&display_type=table)
+ [live-roof-650-townsend-dropouts-zero-baseline](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-dropouts-zero-baseline&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019032221-3-g43895e54&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.1.0-develop-2019032221-3-g43895e54
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.1.0-develop-2019032221-3-g43895e54)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.1.0-develop-2019032221-3-g43895e54/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.1.0-develop-2019032221-3-g43895e54

**Comment by anth-cole** [comment]
only hitl failure was a nap CPU idle failure which this PR would not have affected - think this is good to go @benjaminaltieri you good if I merge?

**Comment by silverjam** [comment]
With this change we don't seem to generating a GGA message anymore:

```
root@piksi:~ # socat unix-connect:/var/run/sockets/nmea_external.pub,type=5 -|grep GGA
... no output ...
```

**Comment by benjaminaltieri** [comment]
@anth-cole could that regression be related to up-revving gnss-converters to v0.3.78?

**Comment by silverjam** [comment]
@anth-cole @nsirola Any ideas?  I think I'm on the latest firmware (which should be generating a pos_llh_cov message).

**Comment by anth-cole** [comment]
I guess it could be - all the tests passed - I did change the input to use the POS_LLH_COV message and not the POS_LLH message

**Comment by anth-cole** [comment]
@silverjam are you getting the other messages? specifically the GST message needs the cov message too

**Comment by benjaminaltieri** [comment]
@silverjam @anth-cole I believe this is a router forwarding issue, need to pass POS_LLH_COV to SBP_PORT_NMEA_BRIDGE

**Comment by silverjam** [comment]
@benjaminaltieri Hah, just noticed this too... testing now.

**Comment by anth-cole** [comment]
ouch - sorry guys, mea culpa my testing was not enough.

File: package/sbp_nmea_bridge/src/sbp_nmea_bridge.c [hunk 1]
```diff
@@ -342,6 +349,16 @@ int main(int argc, char *argv[])
                              SETTINGS_TYPE_FLOAT,
                              notify_soln_freq_changed,
                              &state);
+
+  pk_settings_register_watch(settings_ctx,
```
Comment (nsirola) [inline package/sbp_nmea_bridge/src/sbp_nmea_bridge.c]: btw this should be `pk_settings_register` instead of `pk_settings_register_watch` ?

Comment (nsirola) [reply package/sbp_nmea_bridge/src/sbp_nmea_bridge.c]: Fixing in https://github.com/swift-nav/piksi_buildroot/pull/1205

**Pull request merged**